*** ./node_modules/react-native/packager/react-packager/src/Resolver/polyfills/babelHelpers.js	2017-02-02 16:46:44.840304988 +0530
--- /tmp/babelHelpers.patched.js	2017-02-02 16:31:03.412165656 +0530
***************
*** 18,23 ****
--- 18,28 ----
  // actually, that's a lie, because babel6 omits _extends and createRawReactElement
  
  var babelHelpers = global.babelHelpers = {};
+ babelHelpers.typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) {
+   return typeof obj;
+ } : function (obj) {
+   return obj && typeof Symbol === "function" && obj.constructor === Symbol ? "symbol" : typeof obj;
+ };
  
  babelHelpers.createRawReactElement = (function () {
    var REACT_ELEMENT_TYPE = typeof Symbol === "function" && Symbol.for && Symbol.for("react.element") || 0xeac7;
